package.json

```diff
@@ -1,25 +1,14 @@
 {
   "name": "@tforster/joy",
-  "version": "0.3.3",
+  "version": "1.0.0",
```

Comment:
Let's discuss how we want to manage versioning and when to update to 1.0.0
Some interesting points for discussion:
https://github.com/semver/semver/issues/363